showcase.html

```diff
@@ -128,6 +128,13 @@ <h1>Showcase</h1>
     </header>
 
     <section class="showcase-page">
+        <a class="item" href="https://findanime.vercel.app/">
```

Comment:
Thanks for the submission! Please move the entry to the end and I will go ahead and merge

Reply:
bump @lowzyyy

Reply:
> bump @lowzyyy

Moved the entry